Cargo.toml

```diff
@@ -22,7 +22,9 @@ gtk= "0.8.1"
 glib= "0.9.3"
 libappindicator= "0.5.1"
 
-# [target.'cfg(target_os = "macos")'.dependencies]
-# objc="*"
-# cocoa="*"
-# core-foundation="*"
+[target.'cfg(target_os = "macos")'.dependencies]
+objc = "*"
+objc-foundation = "*"
+objc_id = "*"
+cocoa = "*"
+core-foundation = "*"
```

Comment:
We probably want to have some more strict version requirement for production usage.

Reply:
Fixed in 6db1648